frontend/src/i18n/locales/en.yaml

```diff
@@ -456,9 +457,8 @@ manage:
           no-read-access-to-current: >
             You do not have read access to the video that is currently referenced here.
 
-      titled:
-        title: Title
-        show-title: Show title
+      show-title: Show title
+      show-link: Show button link to video page
```

Comment:
Just "link" instead of "button link" is probably fine, right? Same for German. Of course, especially so if we actually use a normal link design.